operator/go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/openshift/coredns-ocp-dnsnameresolver/operator
 
-go 1.19
+go 1.21
```

Comment:
Can you please also update this in the README as well?

Reply:
Done.